prime-angular-frontend/src/app/modules/health-auth-site-reg/pages/health-auth-care-type-page/health-auth-care-type-page.component.ts

```diff
@@ -63,9 +69,9 @@ export class HealthAuthCareTypePageComponent extends AbstractHealthAuthoritySite
 
   protected patchForm(): void {
     const healthAuthId = +this.route.snapshot.params.haid;
-    const healthAuthSiteId = +this.route.snapshot.params.sid;
-    if (!healthAuthId || !healthAuthSiteId) {
-      throw new Error('No health authority site ID was provided');
+    if (!healthAuthId) {
+      // Don't throw an error as new registrations are created in this view
```

Comment:
Shouldn't there still be an error if no HA ID?

Reply:
It is transferred from vendor page, which was the starting page, to the new starting page.
![image](https://user-images.githubusercontent.com/108699279/199773136-97316979-c0d2-4893-8c9d-a509d085cd79.png)
I was thinking to inherit the existing behavior by doing so.

Reply:
It is transferred from vendor page, which was the starting page, to the new starting page.
![image](https://user-images.githubusercontent.com/108699279/199774055-67ba6a20-202f-4e9b-95db-033a3bfb5a95.png)

I was thinking to inherit the existing behavior by doing so.